python/lsst/ctrl/stats/records/record.py

```diff
@@ -72,14 +83,27 @@ def __init__(self, year, lines):
     def recDatetime(self):
         return parser.parse(self.timestamp)
 
+    def _utcseconds(self, dt):
+        utc_dt = dt.astimezone(tz.tzutc())
+        epoch = datetime.datetime(1970, 1, 1, 0, 0, 0)
+        epoch = epoch.replace(tzinfo=tz.tzutc())
+        return (utc_dt - epoch).total_seconds()
+
+    def _utcDatetime(self, dt):
+        timeFormat = "%Y-%m-%d %H:%M:%S"
+        utc_dt = dt.astimezone(tz.tzutc())
```

Comment:
I checked with the powers that be about this, and legacy code can be either camel-cased (which is what N806 is complaining about) or use the underscore convention, but not both.   (New code is supposed to use underscores).  There were some instances in the tests and a section of new code that used underscores, so I changed those to camel case to be consistent.